.github/workflows/docker-unified.yml

```diff
@@ -423,9 +399,6 @@ jobs:
     name: Build and Push DataHub Elasticsearch Setup Docker Image
     runs-on: ubuntu-latest
     needs: setup
-    outputs:
```

Comment:
Is this just unnecessry? Not providing any value? Or are outputs from prev step just automatically propagated?

Reply:
Ah I see. These are unused, right?